be/src/storage/vectorized/chunk_aggregator.cpp

```diff
@@ -200,34 +209,59 @@ bool ChunkAggregator::_row_equal(const Chunk* lhs, size_t m, const Chunk* rhs, s
     return true;
 }
 
-void ChunkAggregator::update_source(ChunkPtr& chunk) {
-    _is_eq.assign(chunk->num_rows(), 1);
+void ChunkAggregator::update_source(ChunkPtr& chunk, std::vector<RowSourceMask>* source_masks) {
+    size_t is_eq_size = chunk->num_rows();
+    _is_eq.assign(is_eq_size, 1);
```

Comment:
why here should be initialized to 1?

Reply:
It is initially assumed that the keys of all rows are the same.
It should be initialized to 1 when key comparison. like 
`flags[i] = flags[i] & (values[i] == values[i - 1]);`